cmake.deps/CMakeLists.txt

```diff
@@ -12,11 +12,19 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" "${PROJECT_SOURCE_DI
 include(CheckCCompilerFlag)
 include(Util)
 
+set(CMAKE_ARGS_DEFAULT
```

Comment:
```suggestion
set(DEPS_CMAKE_ARGS
```

Reply:
Also changing CMAKE_CACHE_ARGS_DEFAULT variable to DEPS_CMAKE_CACHE_ARGS in that case